services/ows_refactored/agriculture/ows_crop_mask_cfg.py

```diff
@@ -180,13 +180,17 @@
     },
 }
 
-layer = {
-    "title": "Cropland Extent Map for Eastern Africa",
-    "name": "crop_mask_eastern",
-    "abstract": """
+layers = {
+    "title": "Cropland extent map",
+    "abstract": """The Digital Earth Africa cropland extent product  identifies areas of cropping in Africa, using Copernicus Sentinel-2 satellite imagery and machine learning techniques. An accurate, high-resolution, and regularly-updated cropland area map for the African continent is recognised as a useful tool in crop monitoring services. A precise and accurate cropland extent map indicating where cropland occurs serves as a basis for higher-level products such as crop type and watering intensity.""",
+    "layers": [
+        {
+            "title": "Cropland extent map - Eastern Africa",
```

Comment:
now that we only have 2019, can you add 2019 to the product title? otherwise it's not easy for the user to tell which year this is for.